pkg/rancher/migrations.go

```diff
@@ -132,6 +141,33 @@ func createOrUpdateConfigMap(configMapClient controllerv1.ConfigMapClient, cm *v
 	return err
 }
 
+func versionTombstone(configMapController controllerv1.ConfigMapController) error {
+	cm, err := getConfigMap(configMapController, rancherVersionTombstoneConfig)
+	if err != nil || cm == nil {
+		return err
+	}
+
+	if !semver.IsValid(rancherversion.Version) {
+		logrus.Errorf("rancher version %s is not semver compliant, skipping tombstone validation", rancherversion.Version)
```

Comment:
Nit: Might be nice as a dev-ex thing if we can have some indication that this is fine/expected in dev environments:

```suggestion
		logrus.Errorf("rancher version %s is not semver compliant, skipping tombstone validation. This is expected in development enviroments.", rancherversion.Version)
```